README.md

```diff
@@ -82,14 +82,13 @@ need to add the following to `.env`:
 ```
 WEB_CAL_URL=webcal://localhost:8080
 ```
-### Texting (not required)
+### Texting 
 
 Some endpoints will cause the code to send texts to users. In order for this to work, it is important
 that you add the correct environment variables to the `.env` file.
 
 ```
-NEXMO_API_KEY=yournexmoapikey
-NEXMO_API_SECRET=yourapisecret
+AWS_SMS_REGION=yourawsregion (eu-west-1)
```

Comment:
Btw we needed a separate region for AWS SMS because SMS is not supported in many locations. It is only supported from Frankfurt (eu-west-1). However, you can still send to UK mobile numbers.